common/views/components/MainViewerPrototype/MainViewerPrototype.tsx

```diff
@@ -335,11 +335,21 @@ const MainViewer: FunctionComponent<Props> = ({
     let currentCanvas;
     if (firstRenderRef.current) {
       setActiveIndex(canvasIndex);
-      mainViewerRef &&
-        mainViewerRef.current &&
-        mainViewerRef.current.scrollToItem(canvasIndex, 'start');
-      setFirstRender(false);
       currentCanvas = canvases && canvases[canvasIndex];
+      const viewer = mainViewerRef?.current;
+      const isLandscape = currentCanvas.width > currentCanvas.height;
+
+      if (isLandscape && canvasIndex === 0) {
+        const ratio = currentCanvas.height / currentCanvas.width;
+        const renderedHeight = mainAreaWidth * ratio * 0.8;
```

Comment:
0.8 because `80%` is the `max-width` for images within their containing item. This should probably be stored as a variable.